vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -1,109 +1,167 @@
 
-Once authenticated, you'll get the message "No conferences are available" and, if you check your browser console, you should see a `403 (Forbidden)` error for the request made to `localhost:8080/services/conference/api/conferences`. This is expected because we have not yet granted the new role to the admin user. 
+> Note: Once authenticated, the message "No conferences are available" is generated. If you check your browser
+console, you should see a `403 (Forbidden)` error for the request made to `localhost:8080/services/conference/api/conferences`. This is expected because the admin user has not yet been granted the new role. 
 
-Now let's give the admin user the correct role. 
+### Step 4: Login to Keycloak
 
-4. Login to keycloak on <http://localhost:9080> using the `admin/admin` credentials. 
+1. Go to <http://localhost:9080> 
+2. Login using the the default credentials of `username: admin`, `password: admin`
 
-First we need to create the two roles per our requirements. We're going to add the roles to the `internal` client because it's the one configured by default in the Spring Boot application.yml.
+### Step 5: Create the `conference-user` and `conference-admin` roles 
+
+Add the `conference-user` and `conference-admin` roles to the `internal` client. 
           
-5. Go to `Clients → internal → Roles` and click `Add Role`
-6. Fill in the `Role Name` with `conference-admin` and click `Save`
-7. Repeat steps 5-6 to create the `conference-user` role. 
+1. Go to `Clients` → `internal` → `Roles`
+2. Click `Add Role`
+3. Fill in the `Role Name` with `conference-admin`
+4. Click `Save`
+5. Repeat these steps to create the `conference-user` role
+
+> Note: The `internal` client is configured by default in the Spring Boot `application.yml`.
+
+### Step 6: Map the `conference-user` role to the admin user
 
-Now we need to map this role to our user.
+To secure the `getAllConferences` API:
 
-8. Go to `Users → View all users → admin → Role Mappings`
-9. Select `internal` for the `Client Roles` and then move `conference-user` from `Available Roles` to `Assigned Roles`
-10. Go back to the MFE and you should now see the full list of Conferences.
+1. Go to `Users` → `View all users` → `admin` → `Role Mappings`
+2. Select `internal` for the `Client Roles` 
+3. Move `conference-user` from `Available Roles` to `Assigned Roles`
+4. Return to the MFE to confirm you see the full list of Conferences
 
-We've now successfully secured the `getAllConferences` API but we have more to do. The admin user was granted just the `conference-user` role but still has access to delete Conferences. We need to lock that down.
+### Step 7: Restrict the ability to delete Conferences
 
-11. Go back into the `ConferenceResource.java` file and add this annotation to the `deleteConference` method:
+The `conference-user` role grants the admin user permission to delete Conferences. To restrict the delete method to the `conference-admin` role:
```

Comment:
conference-user -> conference-admin

Reply:
i thought we were modifying things so the conference-user couldn't delete but the conference-admin could??